qiskit_experiments/library/characterization/cr_hamiltonian_analysis.py

```diff
@@ -0,0 +1,298 @@
+        curve.SeriesDef(
+            name="z|c=0",
+            fit_func=lambda x, px0, px1, py0, py1, pz0, pz1, b: curve.fit_function.bloch_oscillation_z(
+                x, px=px0, py=py0, pz=pz0, baseline=b
+            ),
+            filter_kwargs={"control_state": 0, "meas_basis": "z"},
+            plot_color="blue",
+            plot_symbol="o",
+            canvas=2,
+        ),
+        curve.SeriesDef(
+            name="x|c=1",
+            fit_func=lambda x, px0, px1, py0, py1, pz0, pz1, b: curve.fit_function.bloch_oscillation_x(
+                x, px=px1, py=py1, pz=pz1, baseline=b
+            ),
+            filter_kwargs={"control_state": 1, "meas_basis": "x"},
+            plot_color="red",
+            plot_symbol="^",
+            canvas=0,
+        ),
+        curve.SeriesDef(
+            name="y|c=1",
+            fit_func=lambda x, px0, px1, py0, py1, pz0, pz1, b: curve.fit_function.bloch_oscillation_y(
+                x, px=px1, py=py1, pz=pz1, baseline=b
+            ),
+            filter_kwargs={"control_state": 1, "meas_basis": "y"},
+            plot_color="red",
+            plot_symbol="^",
+            canvas=1,
+        ),
+        curve.SeriesDef(
+            name="z|c=1",
+            fit_func=lambda x, px0, px1, py0, py1, pz0, pz1, b: curve.fit_function.bloch_oscillation_z(
+                x, px=px1, py=py1, pz=pz1, baseline=b
+            ),
+            filter_kwargs={"control_state": 1, "meas_basis": "z"},
+            plot_color="red",
+            plot_symbol="^",
+            canvas=2,
+        ),
+    ]
+
+    @classmethod
+    def _default_options(cls):
+        """Return the default analysis options."""
+        default_options = super()._default_options()
+        default_options.data_processor = dp.DataProcessor(
+            input_key="counts", data_actions=[dp.Probability("1"), dp.Eigenvalue()]
+        )
+        default_options.curve_plotter = "mpl_multiv_canvas"
+        default_options.xlabel = "Net cross resonance duration"
+        default_options.ylabel = "<X(t)>,<Y(t)>,<Z(t)>"
+        default_options.xval_unit = "s"
+        default_options.style = curve.visualization.PlotterStyle(
+            figsize=(8, 10),
+            legend_loc="lower right",
+            fit_report_rpos=(0.28, -0.10),
+        )
+        default_options.ylim = (-1, 1)
+
+        return default_options
+
+    def _setup_fitting(self, **extra_options) -> Union[Dict[str, Any], List[Dict[str, Any]]]:
+        """Fitter options."""
+        user_p0 = self._get_option("p0")
+        user_bounds = self._get_option("bounds")
+
+        bounds = {
+            "px0": user_bounds["px0"] or (-np.inf, np.inf),
+            "py0": user_bounds["py0"] or (-np.inf, np.inf),
+            "pz0": user_bounds["pz0"] or (-np.inf, np.inf),
+            "px1": user_bounds["px1"] or (-np.inf, np.inf),
+            "py1": user_bounds["py1"] or (-np.inf, np.inf),
+            "pz1": user_bounds["pz1"] or (-np.inf, np.inf),
```

Comment:
Is it intentional to have such large bounds?

Reply:
There is no boundary since Pauli coefficient is device design dependent. Of course we can set some random boundaries, but no reason to choose a specific value.

Reply:
I prefer plus minus inf over something random.